.github/workflows/sync-labels.yml

```diff
@@ -1,16 +1,13 @@
-name: Sync labels from UX Guide repo
+name: Sync labels
 
-on:
- schedule:
-   # daily at 3am UTC so as to probably not disrupt anyone
-   - cron: '0 3 * * *'
+on: [label]
```

Comment:
Instead of running on a schedule, we can run based on label create/modify/delete events. We want them all so we just specify the blanket `label` event.